client/me/account/main.jsx

```diff
@@ -486,29 +501,31 @@ const Account = React.createClass( {
 				</FormFieldset>
 
 				<FormFieldset>
-					<FormLabel htmlFor="url">{ translate( 'Web Address' ) }</FormLabel>
+					<FormLabel htmlFor="user_URL">{ translate( 'Web Address' ) }</FormLabel>
 					<FormTextInput
 						disabled={ this.getDisabledState() }
-						id="url"
-						name="url"
+						id="user_URL"
+						name="user_URL"
 						onFocus={ this.recordFocusEvent( 'Web Address Field' ) }
-						valueLink={ this.valueLink( 'user_URL' ) }
+						value={ this.getUserSetting( 'user_URL' ) || '' }
+						onChange={ this.updateUserSettingInput }
 					/>
 					<FormSettingExplanation>
 						{ translate( 'Shown publicly when you comment on blogs.' ) }
 					</FormSettingExplanation>
 				</FormFieldset>
 
 				<FormFieldset>
-					<FormLabel htmlFor="lang_id">{ translate( 'Interface Language' ) }</FormLabel>
+					<FormLabel htmlFor="language">{ translate( 'Interface Language' ) }</FormLabel>
 					<LanguageSelector
 						disabled={ this.getDisabledState() }
-						id="lang_id"
+						id="language"
 						languages={ config( 'languages' ) }
-						name="lang_id"
+						name="language"
```

Comment:
any particular reason for changing the names of this and other fields? maybe something better for a different PR?